### ISSUE TITLE
Split the Taxonomy Importer

### ISSUE THREAD

**Issue by spficklin** [tripal/tripal#1799]
### Task Description

Currently, the Taxonomy Importer does two things

1. It queries NCBI to pull properties for organisms in Chado
2. It constructs a taxonomic tree using the information from NCBI. 

We should should separate those into two different task.  A user may not want to build a taxonomic tree, just simply load properties for organisms.  

### Branch Name

tv4g4-issue1799-taxonomy_loader_split

**Comment by dsenalik** [comment]
I started working on this, because it has always bugged me that these are combined 😡

There are some severe performance issues with `chado_insert_property` and `chado_insert_dbxref`.
Since these are only used by this importer, this will be a good time to fix it.

*Edit: I think I should make this a separate issue #1925, there are similar slow steps in the GFF importer when creating new terms or DBs which inspired PR #1885

**Comment by dsenalik** [comment]
**Some design notes.**
 - Split into two importers:
![2024-07-15_taximporter](https://github.com/user-attachments/assets/8989b69c-1da2-4e72-a60a-627646c0f0f5)
![2024-07-15_treegenerator](https://github.com/user-attachments/assets/1be8f3c6-87ce-497b-8777-8db6ed0b5604)

 - Only the NCBI taxonomy importer will contact NCBI, the tree generator will use information stored in chado.
 - To be fully backward compatible with the Tripal 3 version, we will need to store, in addition to lineage, the "LineageEx" information. This includes what level each taxon belongs to e.g. "Rutaceae" is a "family", "Magnoliopsida" is a "class", etc. The tripal 3 version got this from the xml and I want to avoid NCBI queries for the tree generator.
 - In order to store this as a property, I need a new cv term for the property "local:lineageex" so unfortunately I will need to include an update hook, but I can recycle 10402 and just run it again as 10406.
 - Running the taxonomy importer once with "Import details for existing species" checked will populate lineageex for an existing site.
 - Minor fix, add the missing space character to the default tree name. I never fixed this in Tripal 3 for backward compatibility, but this is a great time to fix that. i.e. "Site NameTaxonomy Tree" should be "Site Name Taxonomy Tree"
 - Performance is awful, but that will be fixed separately by issue #1925, which is dependent on issue #1780 PR #1844
